roles/solrcloud/templates/solr.in.sh.j2

```diff
@@ -1,51 +1,40 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# By default the script will use JAVA_HOME to determine which java
-# to use, but you can set a specific path for Solr to use without
-# affecting other Java applications on your server/workstation.
-#SOLR_JAVA_HOME=""
+{{ ansible_managed | comment }}
+# Location where the bin/solr script will save PID files for running instances
+# If not set, the script will create PID files in $SOLR_TIP/bin
+SOLR_PID_DIR="/var/run/solr"
+SOLR_HOME="{{ solr_home }}"
+SOLR_DATA_HOME="{{ solr_data_dir }}"
+SOLR_PORT="{{ solr_port }}"
+SOLR_HOST="0.0.0.0"
+SOLR_JETTY_HOST="0.0.0.0"
+# By default the start script uses "localhost"; override the hostname here
+SOLR_HOST="{{ inventory_hostname }}"
+# Location where Solr should write logs to; should agree with the file appender
+# settings in server/resources/log4j.properties
+SOLR_LOGS_DIR="{{ solr_log_dir }}"
 
-# Increase Java Heap as needed to support your indexing / query needs
-SOLR_HEAP='{{ solr_heap }}'
+# Java configuration
+{% if solr_version == '9.2.0' %}
```

Comment:
This seems quite strict. Do we think future 9.x versions will require different Java versions?